2gp-demo/force-app/main/default/classes/LicensingUtil.cls

```diff
@@ -1,49 +1,32 @@
 public with sharing class LicensingUtil {
 	
- 	private static String currentNamespace = getNamespace();
-    private static Set<String> packageNamespaces = new Set<String> {'plpdp_mynamespace1', 'IsvlTdxDemo'};
-    
-	
     public static boolean hasAccessToLCP(String customPerm) {
         // TODO: Confirm that the customPerm is not null, exists, and is a Licensed Custom Perm - otherwise throw exception
         
         /*
         System.debug('debug Test.isRunningTest ' + Test.isRunningTest());
         System.debug('debug isDevOrg ' + isDevOrg());
-        System.debug('debug isPackagingOrg ' + isPackagingOrg());
         System.debug('debug userHasPackageLicense ' + userHasPackageLicense());
         System.debug('debug FeatureManagement.checkPermission ' + FeatureManagement.checkPermission(customPerm));
-        System.debug('debug getNamespace ' + getNamespace());
         System.debug('debug getPackageId ' + getPackageId());
         */
         
         if(Test.isRunningTest() ||                           // bypass access check for automated tests
            isDevOrg() ||                                     // bypass access check if within a developer org (without local namespace)
-           isPackagingOrg() ||                               // bypass access check if within your packaging org (with local namespace)
-           userHasPackageLicense() ||         				 // allow access if the user has a managed package license (for hybrid licensing)
+           //userHasPackageLicense() ||         				 // allow access if the user has a managed package license (for hybrid licensing)
            FeatureManagement.checkPermission(customPerm)) {  // check for user's access for the feature gated by this LCP
             return true;
         }
         
         return false;
     }
     
-    public static String getNamespace() {
```

Comment:
i'm wondering if it's useful to leave this method in the util. even though it won't be used, and only as a reference or as needed

Reply:
If we need it will be in the git repo history, we can get it whenever required. Let me know if you think otherwise

Reply:
sounds good